Api/Data/FaqInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Inchoo\ProductFAQ\Api\Data;
+
+interface FaqInterface
+{
+    const FAQ_ID = 'faq_id';
+    const QUESTION = 'question_content';
+
+    /**
+     * @return int|null
+     */
+    public function getId();
+
+    /**
+     * @codingStandardsIgnoreLine - id cannot be set to particular type
+     * @param mixed $id
+     * @return FaqInterface
+     */
+    public function setId($id);
```

Comment:
ako je return type FaqInterface:
```suggestion
    public function setId($id): FaqInterface;
```

Reply:
postavljen return type